arrow/arrow.py

```diff
@@ -937,23 +948,23 @@ def humanize(
                     years = sign * int(max(delta / 31536000, 2))
                     return locale.describe("years", years, only_distance=only_distance)
 
-            else:
+            elif isinstance(granularity, six.string_types):
```

Comment:
It would be nice to avoid introducing an extra dependency in `six`, maybe there's another way we could do this?

Reply:
We can use the `isstr` function in `utils.py` or we can import it from `future.utils`.

Reply:
updated to use utils.isstr, good catch